src/jsx/import/result/current-result.jsx

```diff
@@ -188,66 +192,64 @@ var CurrentResultDisplay = React.createClass({
         ) {
-          // local variables
-            var resultData = this.state.ajax_retrieval_result
+            // local variables
+            var resultData = this.state.ajax_retrieval_result;
             const status = resultData.status;
 
-          // do not present status
+            // do not present status
             delete resultData.status;
 
-          // generate result
-            var resultList = <ul className='result-list'>{
+            // generate result
+            var resultList = (<ul className='result-list'>{
                 Object.entries(resultData).map(([item_key, value]) =>
-                    <li key={item_key}>{item_key}: {
+                    (<li key={item_key}>{item_key}: {
                         Array.isArray(value) ?
-                            <ul className='sublist' key={'sublist-' + item_key}>
+                            <ul className='sublist' key={`sublist-${item_key}`}>
                                 {
-                                    value.map(function(value, index) {
-                                        return <li key={'subitem-' + index}>{value}</li>;
-                                    })
+                                    value.map((value, index) =>
+                                    <li key={`subitem-${  index}`}>{value}</li>)
                                 }
                             </ul>
-                        : value
+                            : value
                     }
-                    </li>
-                )
-            }</ul>;
+                     </li>))
+            }
+            </ul>);
         } else if (
             resultData &&
             this.props &&
             this.props.results &&
             this.props.results.data &&
             Object.keys(resultData).length > 0
         ) {
-            var resultList = <ul className='result-list'>{
+            var resultList = (<ul className='result-list'>{
                 Object.entries(resultData).map(([item_key, value]) =>
-                    <li key={item_key}>{item_key}: {
+                    (<li key={item_key}>{item_key}: {
                         Array.isArray(value) ?
-                            <ul className='sublist' key={'sublist-' + item_key}>
+                            <ul className='sublist' key={`sublist-${item_key}`}>
                                 {
-                                    value.map(function(value, index) {
-                                        return <li key={'subitem-' + index}>{value}</li>;
-                                    })
+                                    value.map((value, index) =>
+                                    <li key={`subitem-${  index}`}>{value}</li>)
                                 }
                             </ul>
-                        : value
+                            : value
                     }
-                    </li>
-                )
-            }</ul>;
+                     </li>))
```

Comment:
I think the closing parentheses, should be on their own lines, unless you can show me eslint preference for otherwise.